pyproject.toml

```diff
@@ -8,17 +8,16 @@ authors = [
   { name = "Nitrokey", email = "pypi@nitrokey.com" },
 ]
 readme = "README.md"
-requires-python = ">=3.6"
+requires-python = ">=3.9"
```

Comment:
This is worth documenting in the Readme (separate PR perhaps?):
- https://github.com/Nitrokey/pynitrokey/issues/212